logger/logger.go

```diff
@@ -0,0 +1,44 @@
+// Package logger defines the Cloudinary Logger
+package logger
+
+import (
+	"log"
+)
+
+type Level int8
+
+const ERROR Level = 1
+const DEBUG Level = 2
+
+type LoggerFunction func(...interface{})
+type Logger struct {
+	ErrorLogger LoggerFunction
+	DebugLogger LoggerFunction
+	level       Level
+}
+
+func (l *Logger) SetLevel(level Level) {
+	l.level = level
+}
+
+func (l *Logger) Error(v ...interface{}) {
+	l.ErrorLogger(v)
```

Comment:
I would add 
`if l.level >= ERROR {`

So, users could completely disable logging by setting level to 0.

Reply:
Should the description of the logger clarify that it's possible to completely disable logging?

Reply:
Added `logger.NONE` level for disabling logger. Added it to the README